eureka/S3_data_reduction/bright2flux.py

```diff
@@ -183,19 +184,30 @@ def convert_to_e(data, meta, log):
     meta:   MetaClass
         The metadata object.
     """
+    if data.shdr['BUNIT'] != 'ELECTRONS/S':
```

Comment:
If the units are already in electrons/s, why do we need to download any files?

Reply:
The statement is !=, so no files are downloaded if the units are already good.

Reply:
so it is!